Library/SortParams.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recurly
+{
+    public class SortParams
+    {
+        public enum SortField
+        {
+            CreatedAt,
+            UpdatedAt
+        }
+
+        public enum OrderDirection
+        {
+            Asc,
+            Desc
+        }
+
+        public SortField? Sort;
+        public OrderDirection? Order;
```

Comment:
Another possibility is just to name the field the same as the type if that's more descriptive:
    
    public SortField? SortField;
    public SortDirection? SortDirection